### PR TITLE
Fixed autoscaling metrics unit

### PR DESCRIPTION
**Pull request by piosz**

### CODE REVIEW

File: metrics/processors/node_autoscaling_enricher.go [hunk 1]
```diff
@@ -51,13 +51,13 @@ func (this *NodeAutoscalingEnricher) Process(batch *core.DataBatch) (*core.DataB
 			memUsed := getInt(metricSet, &core.MetricMemoryUsage)
 
 			if availableCpu.MilliValue() != 0 {
-				setInt(metricSet, &core.MetricPercentageCpuUsed, (cpuUsed*100)/availableCpu.MilliValue())
-				setInt(metricSet, &core.MetricPercentageCpuRequested, (cpuRequested*100)/availableCpu.MilliValue())
+				setInt(metricSet, &core.MetricNodeCpuUtilization, cpuUsed/availableCpu.MilliValue())
```
Comment (mwielgus) [inline metrics/processors/node_autoscaling_enricher.go]: set float

**Comment by piosz** [comment]
Closing as discussed offline

File: metrics/processors/node_autoscaling_enricher.go (hunk 1 above)
Comment (piosz) [inline metrics/processors/node_autoscaling_enricher.go]: done

**Comment by mwielgus** [comment]
LGTM

**Pull request merged**